tdvt/tdvt/exprtests/standard/expected.setup.cast.str.datetime.txt

```diff
@@ -0,0 +1,130 @@
+<results>
+  <test name='DATETIME(&apos;1900-01-01 01:00:00&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.sqlserver].[TEMP(Test)(3232345613)(0)]</column>
+      </schema>
+      <tuple>
+        <value>#1900-01-01 01:00:00#</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='DATETIME(&apos;1900-12-30 12:00:00&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.sqlserver].[TEMP(Test)(3232345613)(0)]</column>
+      </schema>
+      <tuple>
+        <value>#1900-12-30 12:00:00#</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='DATETIME(&apos;1900-12-31 23:59:59&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.sqlserver].[TEMP(Test)(3232345613)(0)]</column>
+      </schema>
+      <tuple>
+        <value>#1900-12-31 23:59:59#</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='DATETIME(&apos;2018-01-01 01:00:00&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.sqlserver].[TEMP(Test)(3232345613)(0)]</column>
+      </schema>
+      <tuple>
+        <value>#2018-01-01 01:00:00#</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='DATETIME(&apos;2018-12-29 12:00:00&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.sqlserver].[TEMP(Test)(3232345613)(0)]</column>
+      </schema>
+      <tuple>
+        <value>#2018-12-29 12:00:00#</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='DATETIME(&apos;2018-12-31 23:59:59&apos;)'>
+    <table>
+      <schema>
+        <column>[cast_calcs.sqlserver].[TEMP(Test)(3232345613)(0)]</column>
+      </schema>
+      <tuple>
+        <value>#2018-12-31 23:59:59#</value>
+      </tuple>
+    </table>
+  </test>
+  <test name='DATETIME(STR([datetime0]))'>
+    <table>
+      <schema>
+        <column>[cast_calcs.sqlserver].[TEMP(Test)(3072781275)(0)]</column>
+      </schema>
+      <tuple>
+        <value>#2004-07-04 22:49:28#</value>
```

Comment:
Hi @gregoryosgood when I run these tests against sqlserver now this test fails - sqlserver converts the seconds value to `00`. Reproduces across platforms. Wondering how this was generated.